src/core/net/nat64_translator.cpp

```diff
@@ -35,6 +35,9 @@
 
 #if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE
 
+#include "icmp6.hpp"
+#include "ip6_types.hpp"
+#include "udp6.hpp"
```

Comment:
Can we remove these `#include`s? I expect they should already be included by `instance.hpp`.